src/services/cgnOperatorSearchService.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * This service interactsnwith the Bonus API
```

Comment:
This comment is wrong

Reply:
done in https://github.com/pagopa/io-backend/pull/804/commits/7d342ba24b1efeb396af1168d077070de40b0b3e